src/common/components/discussion/index.tsx

```diff
@@ -338,50 +338,45 @@ export const Item = (props: ItemProps) => {
             </div>
             <div className="item-inner">
                 <div className="item-figure">
-                    <div className="d-sm-none" id={`${entry.author}-${entry.permlink}`} onClick={(e) => {onShowProfile(e)}}>{UserAvatar({...props, username: entry.author, size: "medium"})}</div>
-                    <div onMouseEnter={(e) => {onShowProfileAvatar(e)}} onMouseLeave={(e, ) => {onHideProfileAvatar(e)}}>
+                    <div className="btn p-0 d-sm-none" id={`${entry.author}-${entry.permlink}`} onClick={(e) => {onShowProfile(e)}}>{UserAvatar({...props, username: entry.author, size: "medium"})}</div>
                         {ProfileLink({...props, username: entry.author, children: 
```

Comment:
@feruzm Just a suggestion, should we not refactor the `ProfilePreview` component so that is takes a child and shows the miniProfile for that on hover? That way, we'll be able to eradicate many states in many components as well as whenever somebody wants to use it in any place, its just about wrapping the target component with this. If you think it will be a good addition, create a trello card separately so @adilrime17 or I can work on it.

Reply:
@ghazanfar-BrilliantStar yes we should do that, make discussion component as light as possible.